core/create_spoken_forms.py

```diff
@@ -6,12 +6,18 @@
 
 from talon import Module, actions
 
-from .keys.keys import symbol_key_words
 from .numbers.numbers import digits_map, scales, teens, tens
 from .user_settings import track_csv_list
 
 mod = Module()
 
+# Keep support for a handful of symbols.
```

Comment:
Elaborate on why we're keeping these three

Reply:
Done